C/tests/c4Test.hh

```diff
@@ -165,32 +165,40 @@ class TransactionHelper {
 
 #pragma mark - C4TEST BASE CLASS:
 
+#define SkipVersionVectorTest 1
```

Comment:
Could you please wrap this with `#ifndef SkipVersionVectorTest`? That way I can predefine it as 0 in a build setting to override this.

Reply:
sure